app/src/androidTest/java/com/github/orkest/View/auth/SignUpFormTest.kt

```diff
@@ -130,4 +136,25 @@ class SignUpFormTest {
         intended((hasComponent(MainActivity::class.java.name)))
         Intents.release()
     }
+
+    @Test
```

Comment:
Shouldn't these added tests be in the SignInTest not the SignUpFormTest file?

Reply:
Yes you're right, I'll move it